assets/mails/notifications_oauthclients.mjml

```diff
@@ -0,0 +1,26 @@
+{{define "content"}}
+<mj-text mj-class="title content-medium">
+	<img src="https://files.cozycloud.cc/email-assets/stack/icon-archive.png" width="16" height="16" style="vertical-align:sub;"/>&nbsp;
+	{{t "Notifications OAuth Clients Title"}}
+</mj-text>
+<mj-text mj-class="content-medium">
+	{{t "Notifications OAuth Clients Greeting"}}
+</mj-text>
+<mj-text mj-class="content-medium">
+	<p>
```

Comment:
Why the `<p>`? We haven't used that for the other emails, is there something special here?

Reply:
It's just that `<br/>` tags are supposed to have a parent such as `<p>`.

Reply:
The parent can be a `<div>`, which is what `mj-text` translates to

Reply:
This MDN documentation says otherwise but I can remove it if that's bothering you.